### DISCUSSION TITLE
How to modify navbar item links

### DISCUSSION THREAD

**Discussion by 1982606762** [alshedivat/al-folio#1349] [Q&A]
Hi, I just forked the repo and managed to serve it. However, I found there are no further modification instructions in the readme file so I'm a bit confused. I want to change the ref of the blog to my self blog link to make it redirect to an outer link. Also, I want to remove some of the navbar items like teaching and projects. I would appreciate it if anyone could help me with it.

**Comment by george-gca** [comment]
To redirect to your external blog, I believe you can do something like the redirect blog post example. To remove the navbar items simply delete the related files inside `_pages`.

**Comment by george-gca** [comment]
About the redirect, check #806, maybe it'll help

**Comment by gpt-dance** [comment]
For remove some of the navbar items like teaching and projects. You can change the nav to false in blog.md, for example, to remove the blog item in navbar.
